From 1e219f8cf91360e9135c0919838997b6763fb912 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Fri, 15 Jan 2016 03:55:44 -0500
Subject: [PATCH] wr-policy-init_t-get-all-service-status

Allow init_t to get status of all services that have type init_t.

systemd-logind service fails to start w/o this rule.

Upstream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 policy/modules/system/init.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/policy/modules/system/init.te b/policy/modules/system/init.te
index 3587fc9..16dfd0d 100644
--- a/policy/modules/system/init.te
+++ b/policy/modules/system/init.te
@@ -1429,3 +1429,6 @@ allow init_t self:capability2 block_suspend;
 # MLS trusted for lowering/raising the level of files
 mls_file_downgrade(init_t)
 mls_file_upgrade(init_t)
+
+# Get all service status (systemd)
+init_get_system_status(init_t)
-- 
2.7.4

